core/src/main/kotlin/org/rewedigital/dialog/model/google/UserStorageHolder.kt

```diff
@@ -0,0 +1,74 @@
+package org.rewedigital.dialog.model.google
+
+import java.util.*
+
+interface UserStorageHolder {
+    var userStorage: String?
+}
+
+val UserStorageHolder.userData: DataStorage
+get() = DataStorage(this)
+
+class DataStorage(private val userData : UserStorageHolder) : LinkedHashMap<String, String>() {
+    init {
```

Comment:
Why don't you use gson or jackson to parse the userdata object?

Reply:
AFIK both are not a dependency right now and that small JSON is very easy to create by hand.